guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,196 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'actions-and-events'
+is: 'experimental'
+---
+## Actions and Events
+
+Watson Work Services gives your App two patterns for dealing with events, which take place either due to changes in the data model and/or due to actions humans take using a participating client in Watson Work Services such as the Workspace clients. One of these is called **Implicit Action Handler**, the other is called **Explicit Action Handler**.
+
+We are standardizing patterns in our programming model and thus the existing App Configuration flow, though is being kept for backwards compatibility, is being incorporated into this new ClientActionHandler flow. It will introduce a change to how to make your app configurable. Currently, in the [Your Apps section](https://developer.watsonwork.ibm.com/apps) there is a section called **Additional Information** where you enter a configuration URL (for configuring your app in a space) and Terms of Services for your app. With this change we will move Terms of Service to it's own section, and remove the **Additional Information** section. The rest of this guide explains what you will do to set up your app to handle user actions regarding App configuration. If you had already establised an app with configuration, you will not need to do anything to continue working as is, and can update your app as needed. 
+
+There are three main concepts to Actions
+
+| Concept                       | Description       |
+| ----------------------------- |:------------------|
+| **Declare**                   | The developer will declare their app will handle an action, either **/Commands** or **Actions from detected intents** or **Action Fulfillment UI**. |
+| **Action Handling**           | This is the act of handling actions fired based on what the developer has declared above. |
+| **Action User Experience**    | Here we have two ways that actions can be exposed in the User Experience. <br><br>**Implicit** where a Watson Work Services client will handle the actual User Experience rendering and interaction. <br>**Explicit** where your app will be invoked from a Watson Work Services client to inject a User Experience via URL. |
```

Comment:
>that actions can be exposed

This may convey the incorrect assumption of the action being exposed for it to be triggered.... vs handled.  Suggest:

There are two approaches to handle and present a user experience to satisfy an action